### ISSUE TITLE
introduce tests to ensure chart image repository/tags exist

### ISSUE THREAD

**Issue by tobybellwood** [uselagoon/lagoon-charts#187]
Given the complex inter-relations between the chart versions and lagoon versions, we should introduce a test that ensures that all images referenced in the charts are actually pullable.  Currently, there is no output to GitHub Actions if the rollout of a chart fails because of a missing image.
```
Sun, 20 Dec 2020 23:36:26 GMT
Creating namespace 'lagoon-core-kirjn6x79v'...
Sun, 20 Dec 2020 23:36:26 GMT
namespace/lagoon-core-kirjn6x79v created
Sun, 20 Dec 2020 23:51:33 GMT
Error: timed out waiting for the condition
```
There's a pretty neat solution at https://github.com/helm-lab/helm-images/blob/master/images.sh using a dry-run to extract the image refs.

**Comment by smlx** [comment]
Hey @tobybellwood could you provide a link to an example test run with this issue?

**Comment by tobybellwood** [comment]
ha! Most of mine from last week until I figured it out!  Here's one from today - https://github.com/uselagoon/lagoon-charts/runs/1586136229?check_suite_focus=true

**Comment by smlx** [comment]
I'm confused, isn't that error what you want to see?

**Comment by tobybellwood** [comment]
yeah - sorry - it took a lot of hunting to work out what it was - the initial error was that it "timed out".

This isn't a must-have, just a nice to have -I'll think about it a bit!

**Comment by smlx** [comment]
I hunted around in the github action s documentation, couldn't find anything about highlighting logs based on regex.

**Comment by smlx** [comment]
Hey I'm just grooming issues - is this still a problem?
